driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/TestSubscriber.java

```diff
@@ -166,7 +169,7 @@ public Subscription getSubscription() {
      * @throws AssertionError if the sequence of items observed does not exactly match {@code items}
      */
     public void assertReceivedOnNext(final List<T> items) {
-        if (tryMultipleTimes(items.size(), () -> getOnNextEvents().size() != items.size())) {
+        if (!tryMultipleTimes(10, () -> getOnNextEvents().size() == items.size())) {
```

Comment:
This was changed because it didnt work and if items.size() is just 1 then its pretty useless as an eventually consistent check.
This class wasn't used by anything but hadn't been deleted so I just fixed it up.

Reply:
Note: `assertReceivedOnNext` previously wasn't used.